docs/configuration.md

```diff
@@ -1062,11 +1062,14 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.ui.showConsoleProgress</code></td>
-  <td>true</td>
+  <td>false</td>
   <td>
     Show the progress bar in the console. The progress bar shows the progress of stages
     that run for longer than 500ms. If multiple stages run at the same time, multiple
     progress bars will be displayed on the same line.
+    <br/>
+    <em>Note:</em> In shell environment, the default value of spark.ui.showConsoleProgress is true or
+    specified by user.
```

Comment:
Let's remove the last `or specified by user.`. Sounds confusing. LGTM otherwise.

Reply:
OK. I will remove it.